extension/src/contentScript/helpers/setIsConnected.ts

```diff
@@ -1,7 +1,6 @@
-export const embedLyraApi = () => {
-  const url = chrome.extension.getURL("lyraApi.min.js");
+export const setIsConnected = () => {
   const script = document.createElement("script");
-  script.src = url;
+  script.textContent = "window.lyra = true;";
```

Comment:
now instead of putting the whole api lib on window, we're simply setting a flag that will be checked by lyra-api